CMakeLists.txt

```diff
@@ -338,37 +338,13 @@ IF(WANT_SDL)
 	FIND_PACKAGE(SDL2)
 	IF(SDL2_FOUND)
 		SET(LMMS_HAVE_SDL TRUE)
-		SET(LMMS_HAVE_SDL2 TRUE)
 		SET(STATUS_SDL "OK, using SDL2")
```

Comment:
This could simply say "OK" now, since we no longer support SDL1.2.

Reply:
Done with commit https://github.com/LMMS/lmms/pull/7443/commits/3bd77d47ba7e7fadca28d435b5a823c60e1799db.